<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TranscoderOptionsAvAAC</class>
 <widget class="QWidget" name="TranscoderOptionsAvAAC">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>480</width>
    <height>344</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QGroupBox" name="encoder_group">
     <property name="title">
      <string>Encoder</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <widget class="QRadioButton" name="encoder_fast">
        <property name="text">
         <string>Fast search</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="encoder_twoloop">
        <property name="text">
         <string>Two loop searching method</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="encoder_anmr">
        <property name="text">
         <string> ANMR method (experimental)</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="bitrateGroup">
     <property name="title">
      <string>Bitrate</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QRadioButton" name="bitrate_default">
        <property name="text">
         <string>Use default bitrate</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="bitrate_target">
        <property name="text">
         <string>Set target rate</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QGroupBox" name="bitrate_group">
        <property name="title">
         <string/>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QSpinBox" name="bitrate_spinbox">
           <property name="suffix">
            <string> kbps</string>
           </property>
           <property name="minimum">
            <number>8</number>
           </property>
           <property name="maximum">
            <number>320</number>
           </property>
           <property name="singleStep">
            <number>8</number>
           </property>
           <property name="value">
            <number>128</number>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSlider" name="bitrate_slider">
           <property name="minimum">
            <number>8</number>
           </property>
           <property name="maximum">
            <number>320</number>
           </property>
           <property name="value">
            <number>128</number>
           </property>
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>bitrate_slider</sender>
   <signal>valueChanged(int)</signal>
   <receiver>bitrate_spinbox</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>170</x>
     <y>29</y>
    </hint>
    <hint type="destinationlabel">
     <x>445</x>
     <y>24</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>bitrate_spinbox</sender>
   <signal>valueChanged(int)</signal>
   <receiver>bitrate_slider</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>407</x>
     <y>18</y>
    </hint>
    <hint type="destinationlabel">
     <x>191</x>
     <y>29</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
